pkg/scheduler/framework/preemption/preemption.go

```diff
@@ -116,6 +116,8 @@ type Interface interface {
 	// Note that both `state` and `nodeInfo` are deep copied.
 	SelectVictimsOnNode(ctx context.Context, state *framework.CycleState,
 		pod *v1.Pod, nodeInfo *framework.NodeInfo, pdbs []*policy.PodDisruptionBudget) ([]*v1.Pod, int, *framework.Status)
+	// PreemptionScoreFuncs extends a list of score functions to prioritize node victims to be preempted.
+	PreemptionScoreFuncs(ctx context.Context, nodesToVictims map[string]*extenderv1.Victims) []func(node string) int64
```

Comment:
`OrderedScoreFuncs` may read better b/c:

1. the prefix "Preemption" is redundant (same as the package name)
2. "Ordered" can clearly state the custom Score functions are evaluated in an ordered manner.